### PR TITLE
[Modular] Adds two new plushies to the Loadout

### PR DESCRIPTION
**Pull request by SarmentiCampbell**
## About The Pull Request
Two new plushies join the loadout, wooo

## Why It's Good For The Game
Cause plushies, woooh

## Changelog
:cl:
Add: Added two new plushies. 
/:cl:

### CODE REVIEW

File: modular_skyrat/modules/customization/game/objects/items/plushes.dm [hunk 1]
```diff
@@ -135,3 +130,24 @@
...
+
+/obj/item/toy/plush/oleplush
+	name = "Irritable goat plushie"
+	desc = "A plush recreation of a purple ovine. Made with 100% real, all natural wool from the creator herself."
+	icon = 'modular_skyrat/modules/customization/icons/obj/plushes.dmi'
+	icon_state = "plushie_ole"
+	inhand_icon_state = "plushie_ole"
+	attack_verb_continuous = list("headbutts", "plaps")
+	attack_verb_simple = list("headbutt", "plap")
+	squeak_override = list('sound/weapons/punch1.ogg'=1, )
```
Comment (chefferz) [inline modular_skyrat/modules/customization/game/objects/items/plushes.dm]: forgot a space after the =

File: modular_skyrat/modules/customization/game/objects/items/plushes.dm [hunk 2]
```diff
@@ -94,17 +91,15 @@
...
-  squeak_override = list('modular_skyrat/modules/emotes/sound/emotes/mothsqueak.ogg'=1)
-
+	name = "Suspicious moth plushie"
+	desc = "A plushie depicting a certain moth. He probably got turned into a marketable plushie."
+	icon = 'modular_skyrat/modules/customization/icons/obj/plushes.dmi'
+	icon_state = "plushie_duffy"
+	inhand_icon_state = "plushie_duffy"
+	attack_verb_continuous = list("flutters", "flaps", "squeaks")
+	attack_verb_simple = list("flutter", "flap", "squeak")
+	squeak_override = list('modular_skyrat/modules/emotes/sound/emotes/mothsqueak.ogg'=1)
```
Comment (chefferz) [inline modular_skyrat/modules/customization/game/objects/items/plushes.dm]: forgot space after = here too, could fix that

**Comment by Arctaisia** [comment]
Big weh

**Comment by Arctaisia** [comment]
Before merge, as sarmi went to sleep, can someone add the young = 1 tag to the Arcplush to make it unbreedable like Ole's? I'm on mobile, so I can't.

**Review by Gandalf2k15 — APPROVED** [review]

**Pull request merged**